api/stately/apps-queries.ts

```diff
@@ -7,24 +7,14 @@ import { ApiApp as StatelyApiApp } from './generated/index.js';
  * Get all registered apps.
  */
 export async function getAllApps(): Promise<[ApiApp[], ListToken]> {
-  let apps = client.withAllowStale(true).beginList('/apps-1');
+  const apps = client.withAllowStale(true).beginList('/apps-1');
   const allApps: ApiApp[] = [];
-  let token: ListToken | undefined = undefined;
-
-  while (true) {
```

Comment:
The default for `beginList` is to stream the entire result set, so looping and using `continueList` isn't necessary.